haechan45th/src/services/userService.js

```diff
@@ -1,4 +1,8 @@
 const userDao = require('../models/userDao')
+const bcrypt = require("bcrypt");
+const saltRounds = 12;
+const config = process.env;
+const jwt = require('jsonwebtoken');
```

Comment:
```suggestion
const bcrypt = require("bcrypt");
const jwt = require('jsonwebtoken');

const userDao = require('../models/userDao')
```
전역 스코프에 변수를 선언하는 것은 권장되지 않습니다. 이는 코드의 유지 보수성을 낮출 수 있고, 예상치 못한 문제를 발생시킬 수 있기 때문입니다.

전역 변수는 어디서든지 접근할 수 있기 때문에, 코드의 어느 부분에서나 해당 변수를 변경할 수 있습니다. 이는 복잡한 코드에서 변수가 의도치 않게 변경되어 버그가 발생할 수 있는 가능성을 높입니다. 또한 전역 변수가 많아질수록 변수명이 충돌하는 등의 문제가 발생할 가능성이 높아집니다.

또한 특정 부분의 함수를 실행시에만 선언되는 지역 스코프의 변수와는 다르게 전역 스코프의 변수는 항상 메모리를 소비하기 때문에 이러한 부분에서도 좋지 않습니다. 따라서 가급적이면 전역적으로 선언하지 않도록 해주세요 :)

추가적으로 모듈을 import 하는 순서에 대한 컨벤션은 `built-in 모듈` - `외부 모듈` - `custom 모듈` 순서입니다!